src/main/java/com/boggle/BoggleBoardGenerator.java

```diff
@@ -0,0 +1,128 @@
+            int rowMax = boggleAnnotations.findClosestLineNumber(boggleAnnotations.getBottomRightY(), boggleAnnotations.colPixelIncrement);
+            List<Integer> arrayRow = boggleBoardGenerator.printAllInBetween(rowMin, rowMax);
+            List<Integer> arrayCol = boggleBoardGenerator.printAllInBetween(colMin, colMax);
+            boggleBoardGenerator.AllVertexCombinations(arrayRow, arrayCol, boggleAnnotations.description);
+            System.out.println(map);
+        }
+    }
+
+        public static List<EntityAnnotation> returnAnnotationsViaGoogle(){
+        List <EntityAnnotation> realAnnotation = new ArrayList<EntityAnnotation>();
+            // Instantiates a client
+            try (ImageAnnotatorClient vision = ImageAnnotatorClient.create()) {
+
+                // The path to the image file to annotate
+                String fileName = "/Users/natashasinghvi/Documents/boggle/src/main/java/com/boggle/TASHUPhotoboggle.jpeg";
+
+                // Reads the image file into memory
+                Path path = Paths.get(fileName);
+                byte[] data = Files.readAllBytes(path);
+                ByteString imgBytes = ByteString.copyFrom(data);
+
+                // Builds the image annotation request
+                List<AnnotateImageRequest> requests = new ArrayList<>();
+                Image img = Image.newBuilder().setContent(imgBytes).build();
+                Feature feat = Feature.newBuilder().setType(Feature.Type.TEXT_DETECTION).build();
+                AnnotateImageRequest request =
+                        AnnotateImageRequest.newBuilder().addFeatures(feat).setImage(img).build();
+                requests.add(request);
+
+                // Performs label detection on the image file
+                BatchAnnotateImagesResponse response = vision.batchAnnotateImages(requests);
+                List<AnnotateImageResponse> responses = response.getResponsesList();
+
+                AnnotateImageResponse annotateImageResponse = responses.get(0);
+                List<EntityAnnotation> annotations = annotateImageResponse.getTextAnnotationsList();
+                realAnnotation = annotations;
+                return annotations;
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+           return realAnnotation;
+        }
+
+        public static int getWidth() throws IOException {
+            BufferedImage bimg = ImageIO.read(new File("/Users/natashasinghvi/Documents/boggle/src/main/java/com/boggle/TASHUPhotoboggle.jpeg"));
+            int width = bimg.getWidth();
+            return width;
+        }
+
+        public static int getHeight() throws IOException {
+            BufferedImage bimg = ImageIO.read(new File("/Users/natashasinghvi/Documents/boggle/src/main/java/com/boggle/TASHUPhotoboggle.jpeg"));
+            int height = bimg.getHeight();
+            return height;
+        }
```

Comment:
the getWidth and getHeight are 1) inefficient because you try to read the entire file twice and 2) buggy because if you change the file in one and not the other the entire BoardGenerator would get messed up. 
Can you create an ImageSize class that stores width and height and define a function like this instead: `public static ImageSize calculateImageSize(String filename)`